ansible/roles/gpu/defaults/main.yaml

```diff
@@ -1,50 +0,0 @@
----
-# see man lspci (-d)
-nvidia_device_query:
-  - vendor: "10de"
-    class: "0302"
-  - vendor: "10de"
-    class: "0300"
-vault_repo_baseurl: "http://vault.centos.org/{{ os_release_file['content'] |b64decode | regex_search('\\d+\\.\\d+\\.\\d+') }}/os/$basearch/"
```

Comment:
These variables are still used in `ansible/roles/gpu/tasks/nvidia-gpu-CentOS7.yaml`

Reply:
it seems like thats for unsupported centos versions https://github.com/mesosphere/konvoy-image-builder/blob/main/ansible/roles/gpu/tasks/nvidia-gpu-CentOS7.yaml#L31

let me confirm